docs/source/api/notebooks/Notebook_5_GP_solution.ipynb

```diff
@@ -0,0 +1,631 @@
+    "\n",
+    "<img src=\"../../../../tutorial/GP_readout.png\" style=\"margin:auto\" width=\"400\" class=\"center\"/>\n",
+    "\n",
+    "The current version of GraphStorm can not directly perform graph prediction. But as GraphStorm supports node-level prediction, we can use a method called `supernode` to perform graph-level predictions.\n",
+    "\n",
+    "----"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## `super-node` Method Explanation\n",
+    "\n",
+    "Instead of using the `Read-out` operation, we add a new node, called **super node**, to each subgraph, and link all original nodes of the subgraph to it, but not adding reversed edges. With these inbound edges, representations of all original nodes in a subgraph could be easily aggregated to the **super node**. We can then use the **super node** as the representation of this subgraph to perform graph-level prediction tasks. The `super-node` method helps us to turn a graph prediction task into a node prediction task.\n",
+    "\n",
+    "<img src=\"../../../../tutorial/GP_supernode.png\" style=\"margin:auto\" width=\"300\" class=\"center\"/>\n",
+    "\n",
+    "----"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Implementation Ideas\n",
+    "\n",
+    "In order to use the `super-node` method in GraphStorm, we need to implement two extra functions.\n",
+    "\n",
+    "- **Raw Graph Data Conversion**: \n",
+    "    Add a super node to each subgraph in the original batch of subgraphs, and store all of them as one heterogenous graph ready for GraphStorm's graph construction CLIs. After be constructed into GraphStorm's distributed graph format, we can use all GraphStorm's built-in GNN model to perform the `super-node` prediction.\n",
+    "- **Customized GNN Encoder** (Optional):\n",
+    "    Create a customized GNN encoder for specific aggregation method for **super nodes**. This is an optional function as all built-in GraphStorm GNN encoders can aggregate and generate embeddings for **super nodes** naturally. But creating a customized GNN encoder could have fine-granular control of aggregation methods, which might minic the `Read-out` method.\n",
+    "\n",
+    "This notebook will demonstrate the `super-node` method by using GraphStorm APIs and other libraries to implement both functions. This notebook serves as an example of a Graph Classification Solution using GrahpStorm APIs. Users can modify the customer GNN model and implement their own version."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "-----\n",
+    "\n",
+    "## Prerequisites\n",
+    "This notebook assumes the following:\n",
+    "\n",
+    "- Python 3;\n",
```

Comment:
Maybe say >=3.8 ? 0.4 won't support anything older

Reply:
Updated the `Prerequisites` with Notebook 1-4. The current one was copied from the Notebook 0.